src/components/Content/RightPanel/TaskList/TaskList.js

```diff
@@ -234,6 +236,11 @@ class TaskList extends Component {
     rpUpdateTaskListFilter(e.target.value);
   }
 
+  clearFilter = () => {
+    const { rpUpdateTaskListFilter } = this.props;
+    rpUpdateTaskListFilter('');
```

Comment:
@4xdk Here, please also give the text input `focus` so that the user can begin typing immediately.